ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -491,28 +663,62 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
-- as: aws-ipi-public-ipv4-pool-byo-subnet-amd-f28-destructive
+- as: aws-ipi-public-ipv4-pool-byo-subnet-role-amd-f28-destructive
   cron: 17 20 27 * *
   steps:
     cluster_profile: aws-2
     env:
+      ASSIGN_ROLES_TO_SUBNETS: "yes"
       BASE_DOMAIN: origin-ci-int-aws.dev.rhcloud.com
+      SUBNET_ROLES: |-
+        [
+          [
+            {
+              "private": "ControlPlaneInternalLB",
+              "public": "ControlPlaneExternalLB IngressControllerLB BootstrapNode ClusterNode"
```

Comment:
Here `ClusterNode` means the cluster nodes would be placed into the public subnet ?

Reply:
Addressed.
Updated in `aws-ipi-byo-subnets-role-only-public-mini-perm-arm-f14`